### PR TITLE
Adjust PlaidML test tolerances

### PR DESCRIPTION
**Pull request by earhart**

### CODE REVIEW

**Comment by silee2** [comment]
@earhart GPU.sigmoid_bprop_n1c1h4 is failing. Any reason why expected value for sigmoid_bprop_n1c1h4 has been changed in this PR?

**Comment by diyessi** [comment]
The vector of expected values changed in the PR.

**Comment by silee2** [comment]
```

File: test/backend_test.in.cpp [hunk 1]
```diff
@@ -5233,8 +5243,8 @@ NGRAPH_TEST(${BACKEND_NAME}, sigmoid_bprop_n1c1h4)
     auto handle = backend->compile(func);
     handle->call_with_validate({result}, {a, b});
 
-    vector<float> expected{0.196612f, 0.0176627f, 0.196612f, 0.0176627f};
-    EXPECT_TRUE(test::all_close(expected, read_vector<float>(result)));
+    vector<float> expected{0.1966119f, 0.01766273f, 0.1966119f, 0.01766273f};
```
Comment (diyessi) [inline test/backend_test.in.cpp]: Rather than using magic values, let's compute the correct values. sigma=1/(1+exp(-x)) and value=sigma(1-sigma).

**Comment by silee2** [comment]
Replaced magic values with computed values. Hope this passed CI.

**Comment by silee2** [comment]
Unfortunately GPU still fails to pass.

**Pull request merged**